docker-compose.yml

```diff
@@ -28,13 +28,21 @@ x-logging:
   options:
     max-size: 1m
     max-file: "1"
+x-volumes:
+  &default-volumes
+  - .:/app:cached
+  - bundle:/bundle
+  - node_modules:/app/node_modules
+  - packs:/app/public/packs
+  - rails_cache:/app/tmp/cache
 
 services:
   db:
     image: postgres
```

Comment:
Does this default to `:latest`? If so, the article recommends using version numbers.

Reply:
Good catch!  I checked my images with `docker images` and redis, postgres, and fake-s3 are all on latest so I'll add a version pinning them to the tag that corresponds to latest as of today.